### PR TITLE
Update plugin for Elasticsearch 2.4.4

### PR DESCRIPTION
**Pull request by lovemeblender**

### CODE REVIEW

File: src/main/java/org/elasticsearch/index/analysis/GreeklishTokenFilterFactory.java [hunk 1]
```diff
@@ -5,15 +5,18 @@
...
 
 public class GreeklishTokenFilterFactory  extends AbstractTokenFilterFactory {
 	private final int maxExpansions;
 	private final boolean generateGreekVariants;
 
 	@Inject
-	public GreeklishTokenFilterFactory(Index index, @IndexSettings Settings indexSettings, @Assisted String name, @Assisted Settings settings) {
-        super(index, indexSettings, name, settings);
+	public GreeklishTokenFilterFactory(Index index,
+									   IndexSettingsService indexSettings,
```
Comment (bill-kolokithas) [inline src/main/java/org/elasticsearch/index/analysis/GreeklishTokenFilterFactory.java]: Indentation seems off

File: src/test/java/org/elasticsearch/index/analysis/SimpleGreeklishAnalysisTest.java [hunk 2]
```diff
@@ -26,12 +25,14 @@ public void testGreeklishAnalysis() {
...
+                .put(IndexMetaData.SETTING_VERSION_CREATED, Version.CURRENT)
+                .put("path.home", "/")
+                .build();
 
         Injector parentInjector = new ModulesBuilder()
             .add(new SettingsModule(indexSettings),
-                 new EnvironmentModule(new Environment(indexSettings)),
-                 new IndicesAnalysisModule()).createInjector();
+                 new EnvironmentModule(new Environment(indexSettings)))
+                 .createInjector();
```
Comment (bill-kolokithas) [inline src/test/java/org/elasticsearch/index/analysis/SimpleGreeklishAnalysisTest.java]: Should be aligned with the `.add` method above.

File: src/main/resources/es-plugin.properties [hunk 3]
```diff
@@ -1,2 +0,0 @@
-plugin=org.elasticsearch.plugin.analysis.greeklish.GreeklishPlugin
```
Comment (bill-kolokithas) [inline src/main/resources/es-plugin.properties:1]: The file is still present.

**Review by bill-kolokithas — APPROVED** [review]
Few minor things only. 👍

File: src/main/java/org/elasticsearch/index/analysis/GreeklishTokenFilterFactory.java (hunk 1 above)
Comment (lovemeblender) [reply src/main/java/org/elasticsearch/index/analysis/GreeklishTokenFilterFactory.java]: Done, it was looking good in IntelliJ!

File: src/test/java/org/elasticsearch/index/analysis/SimpleGreeklishAnalysisTest.java (hunk 2 above)
Comment (lovemeblender) [reply src/test/java/org/elasticsearch/index/analysis/SimpleGreeklishAnalysisTest.java]: done

File: src/main/resources/es-plugin.properties (hunk 3 above)
Comment (lovemeblender) [reply src/main/resources/es-plugin.properties:1]: deleted

**Pull request closed without merging**